### DISCUSSION TITLE
Unable to generate Json report

### DISCUSSION THREAD

**Discussion by dhanushng88** [getgauge/taiko#2156] [General]
Issue: Plugin [JSON Report] with pid [22558] did not exit after 4.00 seconds. Forcefully killing it.
2021/06/28 00:08:28 Failed to read proto message: unexpected EOF

I am getting this above error even though i have the latest version of Json-report plugin

We are using the below gauge version:
Gauge version: 1.0.7
Commit Hash: ed7b4fd6

Plugins
-------
html-report (4.0.10)
js (2.3.17)
json-report (0.3.3)
reportportal (1.12.0)
screenshot (0.0.1)
xml-report (0.2.3)

Can anyone help me out resolving this issue...Thanks in Advance

**Comment by dhanushng88** [comment]
@zabil @negiDharmendra @NivedhaSenthil Can anyone help with your inputs Please....

**Reply by zabil** [reply]
Please use the latest version of Gauge (1.2.1) and check if you still have the issue.

**Reply by dhanushng88** [reply]
@zabil Even after updating the latest version i am seeing the issue
2021/06/28 01:25:23 Failed to read proto message: unexpected EOF
Plugin [JSON Report] with pid [28759] did not exit after 4.00 seconds. Forcefully killing it.

gauge versions:
Gauge version: 1.2.1
Commit Hash: fa7918b

Plugins
-------
html-report (4.0.12)
js (2.3.17)
json-report (0.3.6)
reportportal (1.12.0)
screenshot (0.0.1)
xml-report (0.2.3)
